src/libmongoc/tests/json/sessions/legacy/server-support.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
This was a missing `sessions` test. It passes as-is, so I added it in while moving the other old test to `legacy`.